packages/recommend-core/src/personalization/getPersonalizationFilters.ts

```diff
@@ -0,0 +1,54 @@
+import { getAffinities } from './getAffinities';
+import { getStrategy } from './getStrategy';
+
+type GetPersonalizationFilters = {
+  userToken: string;
+  region: string;
+  apiKey: string;
+  appId: string;
+};
+
+export const getPersonalizationFilters = async ({
+  userToken,
+  region,
+  apiKey,
+  appId,
+}: GetPersonalizationFilters) => {
+  if (!region || !apiKey || !appId) {
+    throw new Error(
+      `[Algolia Recommend] parameters 'region', 'apiKey' and 'appId' are required to enable personalization.`
```

Comment:
```suggestion
      `[Algolia Recommend] Parameters `region`, `apiKey` and `appId` are required to enable personalization.`
```

Reply:
Although `apiKey` and `appId` aren't explicit params provided by the developer, they're inferred from the client. So the message is misleading. We could have another branch for those.